packages/core/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -531,35 +539,27 @@ export async function zipCode() {
     const zipStartTime = Date.now()
     const sourceFolder = modulePath
     const sourceFiles = getFilesRecursively(sourceFolder, false)
+    const dependenciesFolder: FolderInfo = getDependenciesFolderInfo()
 
     try {
-        installProjectDependencies()
+        copyProjectDependencies(dependenciesFolder)
     } catch (err) {
-        void vscode.window.showErrorMessage(CodeWhispererConstants.installErrorMessage)
-        // open build-logs.txt file to show user error logs
-        const logFilePath = await writeLogs()
-        const doc = await vscode.workspace.openTextDocument(logFilePath)
-        await vscode.window.showTextDocument(doc)
-        throw err
+        // continue in case of errors
     }
```

Comment:
We probably want to emit telemetry also here? That would allow us to catch issues with this approach.

Reply:
On another note, do we not have any preexisting tests for zipping?

Reply:
Telemetry is already being emitted inside the `copyProjectDependencies` method on line 508